cpp/lib/src/MBox.cc

```diff
@@ -152,6 +164,7 @@ MBox::Message MBox::getNextMessage() const {
     }
 
     std::string sender, original_host, subject;
+    int priority(-1);
```

Comment:
since allowed priorities are 1-5 we could also use an unsigned with 0 as default